gravitee-am-ui/src/app/services/gio-license.service.ts

```diff
@@ -0,0 +1,164 @@
+    description:
+      'The Kerberos identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-azure-ad': {
+    utm: 'provider_azure_ad',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-azure-ad.svg',
+    description:
+      'The Azure AD identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-france-connect': {
+    utm: 'provider_france_connect',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-france-connect.svg',
+    description:
+      'The France Connect identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-cas': { // utm_source=oss_am&utm_medium=provider_cas&utm_campaign=oss_am_to_ee_am
+    utm: 'provider_cas',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-cas.svg',
+    description:
+      'The CAS provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  // todo: Missing information
+  'am-idp-gateway-handler-saml': {
+    utm: 'provider_saml',
+    image: 'assets/gio-ee-unlock-dialog/gravitee-ee-upgrade.svg',
+    description:
+      'The SAML 2.0 identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-http-flow': {
+    utm: '',
+    image: 'assets/gio-ee-unlock-dialog/gravitee-ee-upgrade.svg',
+    description:
+      'The HTTP Flow identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  // Resources
+  'am-smtp': { // todo: This resource is missing
+    utm: '',
+    image: 'assets/gio-ee-unlock-dialog/am-smtp.svg',
+    description:
+      'The SMTP resource is part of Gravitee Enterprise. Resources allow you to easily reuse some settings.',
+  },
+  // MFA
+  'gravitee-risk-assessment': {
+    utm: 'mfa_risk_base',
+    image: 'assets/gio-ee-unlock-dialog/gravitee-risk-assessment.svg',
+    description:
+      'Risk-based Multi-factor authentication is part of Gravitee Enterprise. MFA allows you to prompt end-users to process MFA verification after they have been authenticated.',
+  },
+  // SAML2 support
+  'am-idp-saml2': {
+    utm: 'provider_saml',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-saml2.svg',
+    description:
+      'SAML 2.0 IdP support is part of Gravitee Enterprise. SAML 2.0 allows you to exchange security information between online business partners.',
```

Comment:
It seems that SAML2 IdP is here twice ? 

https://github.com/gravitee-io/gravitee-access-management/pull/2746/files#diff-1318d2957edcac6eb56c2c2b990682db96da4e276b12ab684d7b84062f6881a1R80